### ISSUE TITLE
ValueError raised when setting Money amount to 0

### ISSUE THREAD

**Issue by linkleonard** [square/connect-python-sdk#45]
There is a None check in the Money.amount() setter. However, the check is incorrect. It should be a `x is None` instead of `not x`.

https://github.com/square/connect-python-sdk/blob/e7aa929/squareconnect/models/money.py#L73

```py
if not amount:
    raise ValueError("Invalid value for `amount`, must not be `None`")
```

```py
if amount is None:
    raise ValueError("Invalid value for `amount`, must not be `None`")
```

My use case that triggered this was when creating an Order, the client library was failing to parse the returned response from Square, probably at the tax and discount fields, as I did not apply any of those to my order.

**Comment by linkleonard** [comment]
I took a closer look at the swagger repository, and it looks like this is the line in the templates that is responsible for generating this code. I don't know enough about swagger to make this call however.

https://github.com/square/connect-api-specification/blob/ab4e96e/swagger-templates/python/model.mustache#L68

**Comment by alecholmes** [comment]
Thanks for reporting, @linkleonard. I'm taking a look and will release a fixed version of the SDK soon.

**Comment by alecholmes** [comment]
This fix has been merged in #47 and a new SDK release has been issued.
